### ISSUE TITLE
When clicking audio recording file options, the canvas scrolls to the bottom right

### ISSUE THREAD

**Issue by yuxiliu1995** [MicroPad/Web#162]
When clicking audio recording file options, instead of making the menu appear, the canvas scrolls to the bottom right.

![bug demo](https://i.imgur.com/UUjQ6O3.gif)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - microPad Version v3.19.0

**Comment by NickGeek** [comment]
Thanks for the bug report! I ran into that the other day too. I'll see what I can do 🙂

**Comment by SubhamBurnwal** [comment]
I tried clicking the ':' button of the audio note and this happened again. I zoomed out of the board and found the "download" option appears very far below the audio note. So I guess the drawing is happening at the wrong reference and micropad is just scrolling to the center of the note (between note and download button)

**Comment by NickGeek** [comment]
@SubhamBurnwal Yep, that looks like the correct place to fix it if we can. I'm unsure as to what a fix would be. If anybody knows any CSS hacks that would help out I'd be keen.
